### ISSUE TITLE
Integrate Django rest_framework into app "samples"

### ISSUE THREAD

**Issue by spitfire-sidra** [spitfire-sidra/mazu#15]
This might be helpful if someone want to build an application based on mazu.

**Comment by 18z** [comment]
I agree, this may simplify our development. I suggest let's just put UI aside, focus on integrating rest_framework. Polish our application. When it's done. We get back to development of UI

**Comment by spitfire-sidra** [comment]
Agree!

**Comment by spitfire-sidra** [comment]
I am working on this issue now.
